### ISSUE TITLE
Support testcafe@2

### ISSUE THREAD

**Issue by AudunWA** [percy/percy-testcafe#268]
Tescafé jsut released a new major version, which means this package has to update its peer dependency on it:
```json
"peerDependencies": {
    "testcafe": "~1"
  },
```
```
npm ERR! Could not resolve dependency:
npm ERR! peer testcafe@"~1" from @percy/testcafe@1.0.2
npm ERR! node_modules/@percy/testcafe
npm ERR!   dev @percy/testcafe@"1.0.2" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: testcafe@1.20.1
npm ERR! node_modules/testcafe
npm ERR!   peer testcafe@"~1" from @percy/testcafe@1.0.2
npm ERR!   node_modules/@percy/testcafe
npm ERR!     dev @percy/testcafe@"1.0.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by AudunWA** [comment]
This was fixed by #270, could you publish a new release @Robdel12?

**Comment by Robdel12** [comment]
Hey @AudunWA! This shouldn't be an issue anymore with v1.0.3 https://github.com/percy/percy-testcafe/releases/tag/v1.0.3
